rules/windows/process_creation/proc_creation_win_lolbin_wlrmdr.yml

```diff
@@ -18,8 +18,7 @@ detection:
         - Image|endswith: '\wlrmdr.exe'
         - OriginalFileName: 'WLRMNDR.EXE'
     selection_child_cli:
-        CommandLine|contains|all:
-            # Note that the dash "-" can be replaced with a slash "/" (TODO: Use the "windash" modifier when it's introduced)
+        CommandLine|windash|contains|all:
```

Comment:
I'm more `CommandLine|contains|all|windash:` , I have try the conversion is the same .
I think, it is more humain : I want "CommandLine" that "contains all" AND use "windash".

`modified` must be updated as detection will change

Reply:
The VSCode plugin gives differing guidance:

![image](https://github.com/SigmaHQ/sigma/assets/954732/8c3338ae-b0b8-445d-ae8e-e03a2d2ac382)

Reply:
VS don't know the actual purpose of `windash`. That's why VS is giving this recommendation, that is correct.
But here `windash` is just the indicator to show both type of variation for given string.

Reply:
So it doesn't matter to me where to put it, I was just seeking clarification because VSCode plugin differs.

So the consensus is: `CommandLine|contains|all|windash` ?

Reply:
Sounds good, thanks all!